litfx-controls/src/main/java/lit/litfx/controls/covalent/PathPane.java

```diff
@@ -238,6 +240,19 @@ private void handlePositionWindowMouseReleased(MouseEvent mouseEvent) {
         previousLocation = new Point2D(getTranslateX(),getTranslateY());
         System.out.println("released previousLocation: "+ previousLocation);
     }
+
+    /**
+     * This convient function allows any node to be allow user to drag or position
+     * window in the scene. For example the title bar or left accent allows the
+     * user to move the pane(window).
+     *
+     * @param node
+     */
+    private void setupMovePaneSupport(Node node){
+        node.setOnMousePressed(mouseEvent -> handlePositionWindowMousePressed(mouseEvent));
+        node.setOnMouseDragged(mouseEvent -> handlePositionWindowMouseDragged(mouseEvent));
+        node.setOnMouseReleased(mouseEvent -> handlePositionWindowMouseReleased(mouseEvent));
```

Comment:
This will be refactored to be custom events in CovalentPaneEvents. This would be nice for touch surfaces where a user can move pane windows similar to mouse events.